diff --git a/node_modules/git-snapshot/index.js b/node_modules/git-snapshot/index.js
index 45d5b11..b394dca 100644
--- a/node_modules/git-snapshot/index.js
+++ b/node_modules/git-snapshot/index.js
@@ -129,7 +129,7 @@ async function gitSnapshot(argv) {
   } finally {
     // Remove added worktree.
     if (isAddedWorktree) {
-      await git(['worktree', 'remove', '-f', worktreePath], onCwdOpts);
+      await git(['worktree', 'remove', '--force', worktreePath], onCwdOpts);
       await git(['worktree', 'prune'], onCwdOpts);
     }
 
